css/selectors/type.json

```diff
@@ -0,0 +1,58 @@
+{
+  "css": {
+    "selectors": {
+      "type": {
+        "__compat": {
+          "description": "Type selector (<code>elementName</code>)",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/Type_selectors",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": true
+            },
+            "ie_mobile": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
```

Comment:
Please add the namespace subfeature (like for universal selector, it has been added at the same time).

See spec (level 4): Type selectors and universal selectors allow an optional namespace component